Resources/Prototypes/SS220/Entities/Clothing/Head/NavyFieldOfficerCap.yml

```
@@ -0,0 +1,17 @@
+# © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatNavyFieldOfficerCap
+  name: фуражка полевого офицера
+  description: Если видите её на станции, прячьте колени
```

```suggestion
  description: Если видите её владельца на станции, прячьте колени.
```
Звучит более понятно, а то есть фуражка на станции - и что? может он её потерял в пыльном углу